src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -256,14 +261,15 @@ private DiagnosticReportingMode GetReportingModeForUnrecognizedProjects()
                 onPathFailure: reportingMode,
                 onLoaderFailure: reportingMode);
 
-            var buildManager = new ProjectBuildManager(Properties, msbuildLogger);
+            var buildHostProcessManager = new BuildHostProcessManager();
+            await using var _ = buildHostProcessManager.ConfigureAwait(false);
```

Comment:
i'm really confused ny this line.  you're configuring the new object you created above?  which was not a task?

Reply:
It's not a task, but it's IAsyncDisposable; the ConfigureAwait here is ensuring the DisposeAsync() is done ConfigureAwait-falsy. Unfortunately the ConfigureAwait in that case seems to lose the underlying type of the object so this was the only way I could figure out how to actually use it.